osf_tests/test_user.py

```diff
@@ -2125,33 +2123,6 @@ def test_validate_social_profile_websites_empty(self):
         self.user.save()
         assert self.user.social['profileWebsites'] == []
 
-    def test_validate_social_profile_website_many_different(self):
```

Comment:
I don't understand the purspose of this test in this context, this was similar to the citations test, which we skipped.

Reply:
- The failure is caused by Django being more strict on URL encoding, especially for special characters in `username`. Thus, instead of skipping the tests, updated the two URLs with correct encoding.
- See [https://gist.github.com/dperini/729294](https://gist.github.com/dperini/729294) and [https://mathiasbynens.be/demo/url-regex](https://mathiasbynens.be/demo/url-regex), which is where we got all the URLs. This resource doesn't encode characters which is causing problems to others as well.